tests/test_flow/conftest.py

```diff
@@ -121,7 +103,26 @@ def _make_list():
 
 
 @pytest.fixture
-def gcs_builder(builder, tmp_gcs_url_prefix):
+def make_dict(process_manager):
+    def _make_dict():
+        if process_manager is None:
+            return {}
+        else:
+            return process_manager.dict()
+
+    return _make_dict
+
+
+@pytest.fixture
+def gcs_fs(use_fake_gcp, make_dict):
+    if use_fake_gcp:
+        return FakeGcsFs(make_dict)
+    else:
+        return get_gcs_fs_without_warnings()
```

Comment:
Are you using this gcs fs instance anywhere? If not, consider calling this fixture `fake_gcs_fs` and use it only when gcs is faked.

Reply:
Yes, we are using `gcs_fs` in places runs on both real and fake gcs e.g. `instrumented_gcs_fs`.